juntagrico/static/juntagrico/js/initJobs.js

```diff
@@ -14,6 +14,12 @@ define([], function () {
             "regex": true,
             "smart": false
         },
+        "columnDefs": [
+            {
+                "targets": [ 5 ],
```

Comment:
use a class name here so if someone adds new collumns before the one defined it does not get all messed up

Reply:
fixed